CMSgov/mint-app
# [EASI-4080 Converting GQL queries Part 3

# EASI-4080


## Changes and Description

- Remove Apollo since it has been discontinued ([end of life blog post](https://www.apollographql.com/blog/announcing-the-end-of-life-schedule-for-apollo-cli-service-commands))
- Converted the remaining queries and deleted old `queries/*` folder
  - GetModelPlanDocuments
  - UploadNewPlanDocument
  - GetOperationalNeed
  - GetOperationalNeedAnswer
  - GetOperationalNeeds
  - CreateDocumentSolutionLinks
  - CreateOperationalSolution
  - CreateOperationalSolutionSubtasks
  - DeleteDocumentSolutionLink
  - DeleteOperationalSolutionSubtasks
  - GetOperationalSolution
  - GetOperationalSolutionSubtasks
  - GetPossibleOperationalSolutions
  - UpdateCustomOperationalNeed
  - UpdateCustomOperationalNeedById
  - UpdateOperationalSolution
  - UpdateOperationalSolutionSubtasks
  - GetClearanceStatuses
  - PrepareForClearance





## How to test this change

1. Go through the app and verify data gets called, saved, and persists.
2. Verify no errors 
3. Cheer! 🎉 



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
